scripts/sct_maths.py

```diff
@@ -472,7 +506,69 @@ def laplacian(data, sigmas):
     # from scipy.ndimage.filters import laplace
     # return laplace(data.astype(float))
 
+def compute_similarity(data1, data2, fname_out='', metric='', verbose=1):
```

Comment:
In the case of this function, where the complexity is greater than the other 2, I would prefer some documentation.